bin/cylc-restart

```diff
@@ -479,4 +480,9 @@ class restart(scheduler):
 
 
 if __name__ == '__main__':
-    main("restart", restart)
+    try:
+        main("restart", restart)
+    except Exception as exc:
+        if cylc.flags.debug:
+            raise
+        sys.exit(exc)
```

Comment:
Made consistent with the other commands, which only dump a traceback if ```--debug``` is used.